tests/src/case0054.c

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Fine to have a dedicated case test, please supply even:

- an example of usage in doc/examples like example15_transaction_coupling.c for the other extension supplied by Globetom
- extend documentation in Chapter 7 https://www.tiian.org/lixa/manuals/html/ch07.html (I will rename the chapter later), create a new section with your extension to TX Demarcation Specification. File doc/manuals/LIXA_Development_TC.xml   I will refine it later, but please provide explanation and the use cases you think are more interesting and supported by your work. Thanks.

Reply:
I have added an example_16_set_profile.c program.
I added a section to the documentation but I do not have the xslt etc. to get the html. It seems to pass xmllint.

Reply:
It sounds good. I can't test it due to the build error introduced by g_steal_pointer with older versions of glib, so I will keep it open to test it in the near future.